### PR TITLE
Added a test for unretired entities (removing an entry in old-entitie…

### PR DESCRIPTION
**Pull request by cjohns-scottlogic**
…s that was there before)

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Added a test to check for unretired entites.

## Related Tickets & Documents

- Ticket Link
- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [X] Yes
- [ ] No, and this is why: 
- [ ] I need help with writing tests

## [optional] Are there any dependencies on other PRs or Work?

### CODE REVIEW

File: tests/integration/pg_load/test_load.py [hunk 1]
```diff
@@ -101,10 +100,43 @@ def test_make_valid_multipolygon(postgresql_conn, sources):
...
         handle_geometry_collection_check(cursor, source)
     postgresql_conn.commit()
     cursor.close()
+
+
+def test_unretired_entities(postgresql_conn):
+    source = "certificate-of-immunity"
+    table = "old_entity"
+
+    for file in ["exported_old_entity_1.csv", "exported_old_entity_2.csv"]:
```
Comment (eveleighoj) [inline tests/integration/pg_load/test_load.py]: data should be in the test_data folder

File: tests/test_unretired/exported_old_entity_2.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+entity|old_entity|entry_date|start_date|end_date|status|notes|dataset
```
Comment (eveleighoj) [inline tests/test_unretired/exported_old_entity_2.csv]: This data is good but doesn't represent the problem when the old_entity.csv is blank. Should be a test case representing what went wrong. could parameterise this one if needed.

File: task/load.sh [hunk 3]
```diff
@@ -75,6 +75,20 @@ elif [[ $DATABASE_NAME != "entity" ]]; then
...
+    echo "Failed to export data from $DATABASE"
+    exit 1
+  fi
+
+  # If exported_old_entity is blank, check there are no records in the sqlite database.
+  # If there aren't, then replace the file with the headers, so it will be processed.
+  if [ ! -s exported_old_entity.csv ]; then
+    COUNT=`sqlite3 $DATABASE 'SELECT COUNT(*) FROM old_entity'`
+    if [ "$COUNT" -eq "0" ]; then
+      echo "entity|old_entity|entry_date|start_date|end_date|status|notes|dataset" > exported_old_entity.csv
```
Comment (eveleighoj) [inline task/load.sh]: This should work but assumes the schema in old_entity will never change.

**Review by CarlosCoelhoSL — DISMISSED** [review]
lgtm

**Review by eveleighoj — COMMENTED** [review]
Looks good, I am a bit concerned that there are now two databases in this code and it's not always clear which database your connecting to, it might be good to use sqlite and postgres to differentiate the two

**Review by eveleighoj — APPROVED** [review]

**Pull request merged**